workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Partials/Improbable.Coordinates

```diff
@@ -0,0 +1,166 @@
+public static readonly Coordinates ZERO = new Coordinates(0, 0, 0);
+
+public static Vector3d operator -(Coordinates v1, Coordinates v2)
+{
+    return new Vector3d(v1.X - v2.X, v1.Y - v2.Y, v1.Z - v2.Z);
+}
+
+public static Coordinates operator -(Coordinates v1, Vector3d v2)
```

Comment:
These mixed Vector/Coordinate operations feel odd. 
Is there some practical use case that makes sense for subtracting a direction from a position, for example?

Reply:
These were all removed in a later commit

Reply:
In the old paradigm of doing things a vector3d is an offset not a direction though.